docs/try-viam/rover-resources/_index.md

```diff
@@ -15,7 +15,7 @@ description: Get your own Viam rover and set it up.
 ---
 
 {{< alert title="Tip" color="tip" >}}
-If you want a convenient mobile {{% glossary_tooltip term_id="base" text="base"%}} for a variety of robotics projects, you can [order your own Viam rover](https://www.viam.com/resources/rover) now.
+If you want a convenient mobile {{% glossary_tooltip term_id="base" text="base"%}} for a variety of robotics projects, you can now [order your own Viam rover](https://www.viam.com/resources/rover).
```

Comment:
Personal nit. "order" or "buy" followed by "now" is a sales exhortation. Either preceded by "now" is instead a "recently-made-available" advisement. Obviously feel free to disregard this if I am being too pedantic 😅 .